packages/ts/hilla-frontend/src/FluxConnection.ts

```diff
@@ -116,4 +153,33 @@ export class FluxConnection {
     };
     return hillaSubscription;
   }
+
+  /**
+   * Adds a listener for the given event type.
+   *
+   * @param type  the type of event
+   * @param listener  the listener to call when the event occurs
+   */
+  addEventListener(type: string, listener: (event: CustomEvent) => void) {
```

Comment:
Any reasons for not extending `EventTarget`? Supported both in browsers and Node.js even.

Reply:
Tried to do that, seems to work. Not sure if all the uglyness is needed though

Reply:
`implements EventTarget` is probably not needed, but does `extends EventTarget` work and make code more compact?

Reply:
Done